src/components/Modals/Onboarding.vue

```diff
@@ -119,6 +121,7 @@
                   </div>
                 </div>
               </div>
+              <!--
```

Comment:
At this point since we already launched with this, we do not need to be commenting these out. Lets just uncomment this